### PR TITLE
Fix prometheus.exporter.cadvisor.md

### LINKED ISSUE #5620: [DOC] Flow cadvisor example error in documentation
### What's wrong?

[Flow Cadvisor DOC](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.cadvisor/)

Incorrect example

### Steps to reproduce

Example:
```
prometheus.exporter.cadvisor "example" {
  docker     = "unix:///var/run/docker.sock"

  storage_duration = "5m"
}
```

Correct Example:
```
prometheus.exporter.cadvisor "example" {
  docker_host     = "unix:///var/run/docker.sock"

  storage_duration = "5m"
}
```

### System information

_No response_

### Software version

v0.37.3

### Configuration

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by Ferrany1**
Fix example config

#### Which issue(s) this PR fixes

Fixes #5620

#### PR Checklist

- [x] Documentation added

### CODE REVIEW

File: docs/sources/flow/reference/components/prometheus.exporter.cadvisor.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ from `prometheus.exporter.cadvisor`:
 
 ```river
 prometheus.exporter.cadvisor "example" {
-  docker     = "unix:///var/run/docker.sock"
+  docker_host     = "unix:///var/run/docker.sock"
```
Comment (spartan0x117) [inline docs/sources/flow/reference/components/prometheus.exporter.cadvisor.md:97]: I realize this spacing was present in the original as well, but would be good to fix it all at once :smiley: 
```suggestion
  docker_host = "unix:///var/run/docker.sock"
```

**Review by spartan0x117 — APPROVED** [review]
LGTM, just had a suggestion that we can touch the spacing up while we are already making the change!

File: docs/sources/flow/reference/components/prometheus.exporter.cadvisor.md (hunk 1 above)
Comment (Ferrany1) [reply docs/sources/flow/reference/components/prometheus.exporter.cadvisor.md]: That's the reasons why originally I haven't changed it.
Done now

**Review by clayton-cornell — APPROVED** [review]
Looks good

**Comment by Ferrany1** [comment]
> Looks good

Can you help me to understand whats else needed to complete pr? Seems like checks have passed and all approvals received

**Comment by clayton-cornell** [comment]
I'll start the merge here :-)

**Comment by Ferrany1** [comment]
> I'll start the merge here :-)

Thanks

**Pull request merged**